target/ppc/translate/vsx-impl.c.inc

```diff
@@ -2136,6 +2136,11 @@ static int pack_flags_xvi(int acc, int sat)
     return (sat << 1) | acc;
 }
 
+static int pack_masks(int pmsk, int ymsk, int xmsk)
```

Comment:
Move this function to internal.h? And also prefix it with 'ger_', since they're 100% specific to the ger instructions?